rcl/README.md

```diff
@@ -1,6 +1,52 @@
-## rcl
+`rcl` consists of functions and structs (pure C) organized into ROS concepts:
 
-Library to support implementation of language specific ROS Client Libraries.
+- Nodes
```

Comment:
This duplicates the information already contained at docs.ros2.org. Just provide a link to that page instead.